src/MQTTConnector.js

```diff
@@ -87,7 +87,7 @@ class MQTTConnector {
             payload_not_available: 'Offline',
             unique_id: `am43_${device.id}_light_sensor`,
             device: deviceInfo,
-            value_template: '{{value_json[\'light\']}}',
+            position_template: '{{value_json[\'light\']}}',
```

Comment:
Renaming in HA was only done for cover mqtt discovery. So this one is still value template